deploy/kubeturbo-for-openshift/README.md

```diff
@@ -84,10 +84,16 @@ spec:
     kubeturbo: deployable
   containers:
   - name: kubeturbo
-    image: vmturbo/kubeturbo:60os
+    # The image is the same one as used in non-OpenShift k8s distributions
+    image: vmturbo/kubeturbo:60
     args:
-      - --kubeconfig=/etc/kubeturbo/admin.kubeconfig
+      # Need specify the admin.kubeconfig file only for OpenShift version before 3.6
+      # - --kubeconfig=/etc/kubeturbo/admin.kubeconfig
       - --turboconfig=/etc/kubeturbo/config
+      - --k8sVersion=1.6
```

Comment:
Can omit the parameter `k8Version`.  The default value for `k8sVersion` is `1.6.0`.